app/Http/Requests/User/UpdateRequest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Requests\User;
+
+use Carbon\Carbon;
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $minimum_date = Carbon::now()->subYears(18)->toDateString();
+
+        return [
+            'gender' => ['required', Rule::in(['f', 'm', 'o'])],
+            'birthdate' => 'required|date|before:'.$minimum_date,
+            'phone_number' => ['required', 'string', 'regex:/(\+(\d{3}))?\d{9}/'],
```

Comment:
Did you test the regex?

Reply:
Yes I did, and it's the same used in the browser. But should't this be extracted to somewhere to be easily changed?

Reply:
No, it is OK. 👍